SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+We use changeset to provide new version everyday with fixes, features and major release when we have a breaking change.
+Let latest is the better. We also provide [talend-scripts upgrade:deps](./tools/upgrade-deps) scripts to make it easier to be aligned and updated.
+The script comes with option about security use case so please read it first.
+
+Distance to latest from npm is added on [dependencies.txt](./dependencies.txt) file on each upgrade PR.
+
+## Reporting a Vulnerability
+
+Please open an issue on github repository if you are outside Talend or in the companie tracker of your team if you are inside Talend.
+You can then ask for help if you need to address the issue.
```

Comment:
```suggestion
Please open an issue on this github repository if you are outside Talend, or in the company's tracker for your team if you are inside Talend.

You can then ask for help if you need to address the issue.
```

Reply:
Which make me think packages should have codeowners, making "asking for help" (and reviews) easier.

Reply:
This is interesting proposal I would like to see.
> With protected branches enabled, a code owner for each owned file has to leave a review before anyone can merge a pull request to that branch.
But today it is quite hard to have reviews so I am afraid it makes it more complex.
I would love to try, please go ahead with a PR.